journal/modernized/04-chapter-3.adoc

```diff
@@ -70,6 +73,8 @@ that nothing may rule in you but the life, which stands in the Lord God.
 
 G+++.+++ F.
```

Comment:
Oops, I lied! This isn't perfect yet. :) the G+++.+++ F. needs a signature tag above it.  Like this: 

`[.signed-section-signature]`
`G+++.+++ F.`